cmd/podmanV2/images/history.go

```diff
@@ -100,24 +104,50 @@ func history(cmd *cobra.Command, args []string) error {
 	} else {
 		switch {
 		case opts.human:
-			row = "{{slice .ID 0 12}}\t{{humanDuration .Created}}\t{{ellipsis .CreatedBy 45}}\t{{humanSize .Size}}\t{{.Comment}}\n"
+			row = "{{.ID}}\t{{.Created}}\t{{.CreatedBy}}\t{{.Size}}\t{{.Comment}}\n"
 		case opts.noTrunc:
-			row = "{{.ID}}\t{{humanDuration .Created}}\t{{.CreatedBy}}\t{{humanSize .Size}}\t{{.Comment}}\n"
+			row = "{{.ID}}\t{{.Created}}\t{{.CreatedBy}}\t{{.Size}}\t{{.Comment}}\n"
 		case opts.quiet:
 			hdr = ""
 			row = "{{.ID}}\n"
 		}
 	}
 	format := hdr + "{{range . }}" + row + "{{end}}"
 
-	tmpl := template.Must(template.New("report").Funcs(report.PodmanTemplateFuncs()).Parse(format))
+	tmpl := template.Must(template.New("report").Parse(format))
 	w := tabwriter.NewWriter(os.Stdout, 8, 2, 2, ' ', 0)
-
-	_, _ = w.Write(report.ReportHeader("id", "created", "created by", "size", "comment"))
-	err = tmpl.Execute(w, results.Layers)
+	err = tmpl.Execute(w, hr)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "Failed to print report"))
 	}
 	w.Flush()
 	return nil
 }
+
+type historyreporter struct {
```

Comment:
Any reason we can't hang the methods off entities.ImageHistoryLayer?

Reply:
evidently the preference is that it be done like this to keep it encapsulated.